mobile/src/main/java/org/openhab/habdroid/ui/activity/PageConnectionHolderFragment.java

```diff
@@ -278,11 +286,19 @@ private void load() {
             headers.put("X-Atmosphere-tracking-id",
                     mAtmosphereTrackingId != null ? mAtmosphereTrackingId : "0");
 
+            String url = mUrl;
+            if (mCallback.serverCanReturnHiddenWidgets()) {
+                url = Uri.parse(url)
+                        .buildUpon()
+                        .appendQueryParameter("includeHidden", "true")
```

Comment:
Usually the server ignores unknown parameters, so you could always add `includeHidden`.

Reply:
If that's the case, I can probably drop the whole `serverCanReturnHiddenWidgets` infrastructure...